controller/Dockerfile

```diff
@@ -10,7 +10,8 @@ RUN curl -sSL -o /usr/local/bin/etcdctl https://s3-us-west-2.amazonaws.com/opdem
     && chmod +x /usr/local/bin/etcdctl
 
 # install confd
-RUN curl -sSL -o /usr/local/bin/confd https://github.com/kelseyhightower/confd/releases/download/v0.8.0/confd-0.8.0-linux-amd64 \
+ENV CONFD_VERSION 0.9.0
```

Comment:
with the switch to parallel layer downloads, things are faster on pull these days so it's not much of an issue. I was just doing this to reduce typing of `0.9.0` to one place but I'd be happy to if this is a concern :)